static/app/components/confirm.tsx

```diff
@@ -315,6 +315,7 @@ class ConfirmModal extends Component<ModalProps, ModalState> {
             ) : (
               <Button
                 onClick={this.handleClose}
+                autoFocus
```

Comment:
This is a pretty big change that probably should not be part of this PR

Reply:
What if we add a new parameter to the confirm modal `isDangerous` which would cause the focus to be on the cancel instead of confirm.